### PR TITLE
proc-nommu: get stack size from program's elf header

### PR DESCRIPTION
**Pull request by HBuczynski**

### CODE REVIEW

File: proc/process.c [hunk 1]
```diff
@@ -559,6 +560,10 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	hal_memset(reloc, 0, sizeof(reloc));
 
 	for (i = 0, j = 0, phdr = (void *)ehdr + ehdr->e_phoff; i < ehdr->e_phnum; i++, phdr++) {
+
+		if (phdr->p_type == PT_GNU_STACK)
+			gnustacksz = phdr->p_memsz;
```
Comment (agkaminski) [inline proc/process.c:565]: Align this size to page size, just in case.

File: proc/process.c [hunk 2]
```diff
@@ -546,6 +546,7 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	char *snameTab;
 	ptr_t *got;
 	struct _reloc reloc[5];
+	size_t stacksz, gnustacksz = 0;
```
Comment (agkaminski) [inline proc/process.c:549]: `size_t stacksz = SIZE_USTACK`, then we don't need `gnustacksz ` variable

File: proc/process.c (hunk 1 above)
Comment (HBuczynski) [reply proc/process.c]: Done

File: proc/process.c (hunk 2 above)
Comment (HBuczynski) [reply proc/process.c]: Done

**Pull request merged**